es/about/index.md

```diff
@@ -34,15 +34,13 @@ conferencias relacionadas a Ruby.
 
 Ruby-Talk, la [lista de correo](/es/community/mailing-lists/) más
 importante sobre el lenguaje Ruby ha crecido hasta lograr un promedio de
-200 mensajes por día.
+200 mensajes por día en el 2006. Se ha reducido en los últimos años a medida que el tamaño de la comunidad movió la discusión de una lista central a muchos grupos más pequeños.
 
 El [índice TIOBE][tiobe], que mide el crecimiento de los lenguajes de
-programación, ubica a Ruby en la posición #13 del ranking mundial.
-Refiriéndose a su crecimiento, predicen, “Todo indica que Ruby llegará a
-estar entre los 10 primeros en menos de 6 meses”. Gran parte de su
-crecimiento se atribuye a la popularidad alcanzada por aplicaciones
-desarrolladas con Ruby, en particular el framework de desarrollo web
-[Ruby on Rails][ror].
+programación, ubica a Ruby entre los diez mejore del ranking mundial.
+Gran parte del crecimiento se atribuye a la
+popularidad del software escrito en Ruby, particularmente el
+[Ruby on Rails] [ror] marco web.
```

Comment:
Place marco web before [Ruby on Rails] and either use tha translation already available in Wikipedia for Web Framework  (Framework para aplicaciones web  https://es.wikipedia.org/wiki/Framework_para_aplicaciones_web ) or propose edition in that Wikipedia page with Marco web as synonim.